MediaBrowser.Model/Net/MimeTypes.cs

```diff
@@ -17,115 +17,135 @@ public static class MimeTypes
-            { ".tiff", "image/tiff" },
-            { ".webp", "image/webp" },
-            { ".ico", "image/vnd.microsoft.icon" },
             { ".svg", "image/svg+xml" },
             { ".svgz", "image/svg+xml" },
+            { ".tbn", "image/jpeg" },
+            { ".tif", "image/tiff" },
+            { ".tiff", "image/tiff" },
+            { ".webp", "image/webp" },
 
             // Type font
             { ".ttf" , "font/ttf" },
             { ".woff" , "font/woff" },
+            { ".woff2" , "font/woff2" },
 
             // Type text
             { ".ass", "text/x-ssa" },
             { ".ssa", "text/x-ssa" },
             { ".css", "text/css" },
             { ".csv", "text/csv" },
+            { ".rtf", "text/rtf" },
             { ".txt", "text/plain" },
             { ".vtt", "text/vtt" },
 
             // Type video
-            { ".mpg", "video/mpeg" },
-            { ".ogv", "video/ogg" },
-            { ".mov", "video/quicktime" },
-            { ".webm", "video/webm" },
-            { ".mkv", "video/x-matroska" },
-            { ".wmv", "video/x-ms-wmv" },
-            { ".flv", "video/x-flv" },
-            { ".avi", "video/x-msvideo" },
-            { ".asf", "video/x-ms-asf" },
-            { ".m4v", "video/x-m4v" },
-            { ".m4s", "video/mp4" },
             { ".3gp", "video/3gpp" },
             { ".3g2", "video/3gpp2" },
+            { ".asf", "video/x-ms-asf" },
+            { ".avi", "video/x-msvideo" },
+            { ".flv", "video/x-flv" },
+            { ".mp4", "video/mp4" },
+            { ".m4s", "video/mp4" },
+            { ".m4v", "video/x-m4v" },
+            { ".mpegts", "video/mp2t" },
+            { ".mpg", "video/mpeg" },
+            { ".mkv", "video/x-matroska" },
+            { ".mov", "video/quicktime" },
             { ".mpd", "video/vnd.mpeg.dash.mpd" },
+            { ".ogg", "video/ogg" },
+            { ".ogv", "video/ogg" },
             { ".ts", "video/mp2t" },
-            { ".mpegts", "video/mp2t" },
+            { ".webm", "video/webm" },
+            { ".wmv", "video/x-ms-wmv" },
 
             // Type audio
-            { ".mp3", "audio/mpeg" },
-            { ".m4a", "audio/mp4" },
             { ".aac", "audio/mp4" },
-            { ".webma", "audio/webm" },
-            { ".wav", "audio/wav" },
-            { ".wma", "audio/x-ms-wma" },
-            { ".ogg", "audio/ogg" },
-            { ".oga", "audio/ogg" },
-            { ".opus", "audio/ogg" },
             { ".ac3", "audio/ac3" },
+            { ".ape", "audio/x-ape" },
             { ".dsf", "audio/dsf" },
-            { ".m4b", "audio/m4b" },
-            { ".xsp", "audio/xsp" },
             { ".dsp", "audio/dsp" },
             { ".flac", "audio/flac" },
-            { ".ape", "audio/x-ape" },
+            { ".m4a", "audio/mp4" },
+            { ".m4b", "audio/m4b" },
+            { ".mid", "audio/midi" },
+            { ".midi", "audio/midi" },
+            // There's also audio/x-midi
```

Comment:
Why not add it then?

Reply:
None of the mime types correlate an extension with two mime types, I didnt know if it could cause a logic issue so I left the comment and didnt add the mime types.
```suggestion
            { ".mid", "audio/x-midi" },
            { ".midi", "audio/x-midi" },
```

Reply:
Ah, in that case I would write out the explanation and leave it alone then.

Reply:
Done